content/handbook/product/ux/research-operations/experience-research-participation.md

```
@@ -0,0 +1,137 @@
+- **ユーザビリティテスト**: GitLab またはプロトタイプデザインを操作しながら考えを声に出していただくセッションで、何がうまく機能し、何がうまく機能しないかを理解する助けとなります。
+- **アンケート**: 経験や好み、特定のトピックに関するフィードバックについてのオンライン質問票です。
+- **モデレーターなしテスト**: 機能のテストやデザインへのフィードバック提供など、ご自身のペースで完了していただくセルフ実施型のアクティビティです。
+- **ダイアリー／縦断研究**: 数週間にわたるリサーチで、初回インタビュー、定期的なアンケート、最終インタビューなど、複数のアクティビティに参加していただきます。実世界のコンテキストの中で、あなたの体験がどのように変化していくかを理解するのに役立ちます。
+
+## 謝礼とインセンティブ
+
+私たちはあなたの時間と専門性を尊重しています。ユーザーインタビューやユーザビリティテストへの参加者には、ギフトカードという形でお礼の品をお贈りしており、通常はセッション終了後 1〜2 営業日以内に処理されます。ギフトカードは RallyUXR を介して Tremendous から送付され、お住まいの地域に応じてさまざまな報酬オプションがあります。これらのメールも私たちの送信エイリアス **research@rally.gitlab.com** から送信されます。これらのお礼メールはメインの受信トレイではなく、プロモーションフォルダに振り分けられることがあるため、メールを期待している場合は両方を確認してください。
+
+ギフトカードを受け取れないが参加を希望される場合は、セッション中にお知らせいただければ処理しません。
+
+**インセンティブについて質問がありますか？** [FAQ セクション](#faqs)をご覧いただくか、下記の[お問い合わせ先](#contact-us)からご連絡ください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)